### DISCUSSION TITLE
Semester

### DISCUSSION THREAD

**Discussion by bayupriyambada** [deaafrizal/cuyuniverse-forum#214] [General Topic]
Kapan akan dikembangkannya a?

**Comment by deaafrizal** [comment]
masih research dulu untuk API youtube nya bro masih blum ada waktu gue.
sementara itu mesti research juga relasi antar tabel antara SESI, SEMESTER dan USERS

**Reply by bayupriyambada** [reply]
Kalau semester kan diawali duluan, lanjut ke sesi setiap semester. di choose user per sesi

**Comment by bayupriyambada** [comment]
Kayaknya referensi bisa dari rctiplus a , kek gini https://www.contohdot.com/semester/1/sesi/1/belajar-coding semisalnya
